ddtrace/contrib/bottle/trace.py

```diff
@@ -30,10 +29,9 @@ def wrapped(*args, **kwargs):
             resource = "{} {}".format(request.method, route.rule)
 
             # Propagate headers such as x-datadog-trace-id.
-            if self.distributed_tracing:
-                context = HTTPPropagator.extract(request.headers)
-                if context.trace_id:
-                    self.tracer.context_provider.activate(context)
+            trace_utils.activate_distributed_headers(
```

Comment:
Same with bottle for the config object 🤔

Reply:
Bottle also doesn't seem to explicitly create its own integration config with distributed_tracing declared 🤕